gender_analysis/corpus.py

```diff
@@ -35,7 +35,7 @@ class Corpus:
     """
 
     def __init__(self, path_to_files, name=None, csv_path=None,
-                       pickle_on_load=None, guess_author_genders=False):
+                       pickle_on_load=None, guess_author_genders=False, ignore_warnings = False):
```

Comment:
Just a heads up, you'll probably run into a merge conflict here because `guess_author_genders` has been removed on the main branch.